### PR TITLE
Add custom name in Navbar

### LINKED ISSUE #917: navbar-brand text should be customizable (not always site.Title)
Currently there is no way to customize navbar-brand text (https://github.com/hugo-toha/toha/blob/a69388d9b8baac3eb13d59d53391811eb6c0b649/layouts/partials/navigators/navbar.html#L66). It will always be site.Title. It would be good if it can be different than that.

The alternative is to replace it manually after generation. Not perfect solution, but it works.

### PR DESCRIPTION
**Pull request by BernatBC**
### Issue

Fixes #917

### Description
If specified in `site.Params.navBarTitle`, that will be the name appearing in the navbar. If the parameter is not given, the value will be `site.Title`.
**Update:** In order to have a different value for each language, the variable has been moved to `data/<language-code>/site.yaml`.

Example config in `data/en/site.yaml`:
```bash
navBarTitle: "Custom Name"
```

### Test Evidence
`navBarTitle: "Custom name"`

image

### CODE REVIEW

**Comment by klugier** [comment]
Looks good to me! Thanks for the fix! Any chances to merge it to the main branch? How should approve it? Moreover, I have one question related to code can navBarTitle be multi-language variable or it is one global variable per project.

**Comment by BernatBC** [comment]
Hi @klugier, we'll have to wait for @hossainemruz. I'm just a user of this theme that sometimes contributes to it.

Regarding to your question, right now it's a global variable located in `config.yaml` file. In order to be a per-language variable, we can put it in `data/<language-code>/site.yaml`.

**Comment by klugier** [comment]
I think we should, it should be per language string not the global one. I can imagine I have different title per language. For example I just want to have "My space", but in my origin language (Polish) I would like to have "Moja przestrzeń".

**Comment by BernatBC** [comment]
Good point @klugier. I'll change it soon.

**Comment by BernatBC** [comment]
I've just moved the variable to `data/<language-code>/site.yaml`. You can change the name by adding:

```yaml
navBarTitle: "Name"
```

File: layouts/partials/navigators/navbar.html [hunk 1]
```diff
@@ -147,4 +152,4 @@
   {{ if $invertedLogo }}
     <img src="{{ $invertedLogo }}" class="d-none" id="inverted-logo" alt="Inverted Logo">
   {{ end }}
-</nav>
+</nav>
```
Comment (klugier) [inline layouts/partials/navigators/navbar.html:155]: Cosmetic: No new line at the end of the file.

**Comment by klugier** [comment]
Hello @hossainemruz, Any changes to merge this change and create new release?

**Comment by hossainemruz** [comment]
Hi @klugier! Sorry, for the delay. I was pretty busy last month. I am already in the process of reviewing the pending PRs. Hopefully, a new release in next week.

**Review by hossainemruz — APPROVED** [review]

**Comment by klugier** [comment]
Thanks! That's good news! Glad that you merged this change.

**Pull request merged**